client/http_sender.go

```diff
@@ -123,22 +120,20 @@ func (s *httpSender) post(call proto.Call) error {
 
 		resp, err := s.client.Do(req)
 		if err != nil {
-			return retry.Continue, err
+			continue
```

Comment:
That said, we should probably cast it to a `net.Error` and check if it's temporary before retrying.